pkg/agent/openflow/network_policy.go

```diff
@@ -731,6 +734,15 @@ func (c *clause) addConjunctiveMatchFlow(featureNetworkPolicy *featureNetworkPol
 				changeType: insertion,
 			}
 		}
+	} else if context.dropFlowEnableLogging != enableLogging {
```

Comment:
I assume this will also work if the namespace annotation is changed while the antrea agent is down, as this code will be invoked upon agent restart. Is that correct?

Reply:
Actually there is another pathway for agent restart [BatchInstallPolicyRuleFlows](https://github.com/antrea-io/antrea/blob/8446156bf8e58b1e8dca0a9d94fc7bec9566f03c/pkg/agent/openflow/network_policy.go#L1219) -> [addRuleToConjunctiveMatch](https://github.com/antrea-io/antrea/blob/8446156bf8e58b1e8dca0a9d94fc7bec9566f03c/pkg/agent/openflow/network_policy.go#L1154) -> [addActionToConjunctiveMatch](https://github.com/antrea-io/antrea/blob/8446156bf8e58b1e8dca0a9d94fc7bec9566f03c/pkg/agent/openflow/network_policy.go#L1180) especially for batch install, which is triggered upon Antrea agent restart.